tests/beaker_tests/.rubocop.yml

```diff
@@ -0,0 +1,4 @@
+# Rubocop configuration
+
```

Comment:
You need to add `inherit_from: ../.rubocop.yml` so that the configuration in the parent file is inherited into this directory. Once you make that change, Travis-CI should be passing for this PR.